server/liveblog/syndication/tasks.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from werkzeug.datastructures import FileStorage
+from superdesk.celery_app import celery
+from superdesk import get_resource_service
+from superdesk.metadata.item import ITEM_TYPE, CONTENT_TYPE
+from .exceptions import APIConnectionError, DownloadError
+from settings import SYNDICATION_CELERY_MAX_RETRIES, SYNDICATION_CELERY_COUNTDOWN
+
+
+logger = logging.getLogger('superdesk')
+
+
+@celery.task(bind=True)
+def send_post_to_consumer(self, syndication_out, producer_post, action='created'):
+    """Send blog post updates to consumers webhook."""
+    from .utils import extract_post_items, extract_producer_post_data
+    consumers = get_resource_service('consumers')
+    items = extract_post_items(producer_post)
+    try:
+        consumers.send_post(syndication_out, {
+            'items': items,
+            'producer_post': extract_producer_post_data(producer_post)
```

Comment:
The call to the function should be before the call to send_post, I think, as it is with the items, just to make the code more readable and easier to debug